packages/ckeditor5-restricted-editing/docs/features/restricted-editing.md

```diff
@@ -86,13 +93,17 @@ ClassicEditor
 	.create( document.querySelector( '#editor' ), {
 		licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
 		plugins: [ RestrictedEditingMode, /* ... */ ],
-		toolbar: [ 'restrictedEditing', /* ... */ ]
+		toolbar: [ 'restrictedEditing:dropdown', /* ... */ ]
 	} )
 	.then( /* ... */ )
 	.catch( /* ... */ );
 ```
 </code-switcher>
 
+<info-box note>
+	Please note there are available toolbar items for inline, block, and both types of editable fields. Read more in the [Configuring the tollbar](#configuring-the-toolbar) section.
+</info-box>
```

Comment:
This probably should be at the end of "### Running the standard editing mode" section, as this applies only to it.

Reply:
Moved it.